modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApi.mustache

```diff
@@ -15,23 +15,25 @@ import javax.validation.constraints.*;
 {{#operations}}
 public class {{classname}}ControllerImp {{#useInterfaces}}implements {{classname}}ControllerImpInterface{{/useInterfaces}} {
 
+    {{#examples}}
     private final ObjectMapper mapper;
 
     @Inject
     private {{classname}}ControllerImp() {
         mapper = new ObjectMapper();
     }
+    {{/examples}}
 
 {{#operation}}
     {{#useInterfaces}}@Override{{/useInterfaces}}
     public {{>returnTypes}} {{operationId}}({{#allParams}}{{>pathParams}}{{>queryParams}}{{>bodyParams}}{{>formParams}}{{>headerParams}}{{#hasMore}}, {{/hasMore}}{{/allParams}}) {{#handleExceptions}}throws Exception{{/handleExceptions}} {
         //Do your magic!!!
-        {{#examples}}
-        String accept = request().getHeader("Accept");
+        {{!examples}}
```

Comment:
Did you mean {{^examples}} ?? I never saw the use of "!" in mustache file but maybe it's a function that exist that I'm not aware of :)